metabase/metabase
# Clean up database guide usability during configuration

Product doc:  https://www.notion.so/metabase/Clean-up-database-guide-usability-during-configuration-65e1ddb6e2464d44bb4b196252e7c661

A new card for showing database help messages.

<img width="1376" alt="Screenshot 2022-01-13 at 16 33 45" src="https://user-images.githubusercontent.com/8542534/149344880-e0d1c733-165e-4442-beb3-7b56c31b3763.png">
<img width="1241" alt="Screenshot 2022-01-13 at 16 36 31" src="https://user-images.githubusercontent.com/8542534/149344891-625d7772-8c4c-46d5-b1a2-d9adc017e2e4.png">

How to test setup changes:
- Run a fresh Metabase instance
- During setup, select a database. You should see the help card. It should be possible to click it.

How to test admin changes:
- Run Metabase
- Go to Admin > Databases > Add database
- The help card should be visible.
